_data/sites.json

```diff
@@ -7718,6 +7718,15 @@
             "yummly.com"
         ]
     },
+    
+    {
+        "name": "Zapier",
+        "url": "https://zapier.com/app/settings/delete",
+        "difficulty": "medium",
```

Comment:
Why is it medium? Normally medium ones require additional steps, it would be good to describe those in the `notes` field.